cpp/include/cugraph/prims/count_if_e.cuh

```diff
@@ -16,7 +16,7 @@
 #pragma once
 
 #include <cugraph/experimental/graph_view.hpp>
-#include <cugraph/prims/edge_op_utils.cuh>
+#include <cugraph/prims/property_op_utils.cuh>
```

Comment:
Is this include necessary?

Reply:
This file is using `cast_edge_op_bool_to_integer` which is in `property_op_utils.cuh` but that file itself is included in `transform_reduce_e.cuh` so I guess it can be removed.

Reply:
> This file is using cast_edge_op_bool_to_integer which is in property_op_utils.cuh

I didn't realize this, and if it's the case, better directly include this file than relying on implementation details of transform_reduce_e.cuh.